tests/unit/UI/Alert/Toast.spec.js

```diff
@@ -0,0 +1,9 @@
+import { mount } from "@vue/test-utils";
+import Toast from "@/components/UI/Alert/Toast";
+
+describe("Toast.vue", () => {
+  it("should render with default values", () => {
+    const wrapper = mount(Toast);
+    expect(wrapper).toBeTruthy();
```

Comment:
this just checks that it is not empty or something right?

Reply:
Also, because if you set values, it overrides default values and those statements doesn't get executed. So code coverage decreases. So we have one test case at least when all of the default statements are executed.